CONTRIBUTING.md

```diff
@@ -266,7 +266,7 @@ for review, which allows you to sidestep this issue.
 This is not a bulletproof method though, as OfBorg still does review requests even on draft PRs.
 ```
 
-## How to backport pull requests
+## How to backport pull requests<a name="how-to-backport-pull-requests"></a>
```

Comment:
```suggestion
## How to backport pull requests <a name="how-to-backport-pull-requests"></a>
```
When I add a space as suggested above then `pandoc --to gfm` adds a `-` at the end of the automatic `id` of the `h2` tag. I assume pandoc behaves like github in this manner.

Reply:
If i add the anchor below the title, then it gets put inside the `p` block